index.html

```diff
@@ -189,7 +225,13 @@ <h3 class="six-title">Profesional Art Printing Data</h3>
             <li class="six-btn">Bootstrap</li>
             <li class="six-btn">Ruby</li>
           </ul>
-          <button class="six-project-btn for-hover for-pressed" type="button">See Project</button>
+        <a><button
+            class="six-project-btn for-hover for-pressed"
+            type="button"
+            data-position="6"
+          >
+            See Project
+          </button></a>
```

Comment:
- [ ] According to you previouse reviewer,  I would kindly suggest that you kindly ensure the buttons in the popup are links which can be visited to view the project's live version or github repository.this is so important for proper user friendly please fix it.
- [ ] see screen rec of your website

https://github.com/mryeminthu/Portfolio/assets/92458236/d84884a5-8ae3-45fc-9c9d-f0cf5f499d66